velox/functions/prestosql/window/tests/WindowTestBase.cpp

```diff
@@ -133,7 +150,11 @@ void WindowTestBase::testTwoColumnOverClauses(
   doubleInput.insert(doubleInput.end(), input.begin(), input.end());
   doubleInput.insert(doubleInput.end(), input.begin(), input.end());
   createDuckDbTable(doubleInput);
-  testWindowFunction(doubleInput, windowFunction, overClauses);
+  if (frameClause.size()) {
```

Comment:
Can't frameClause be sent directly to testWindowFunction ? Why do we check the cases here ? Same comment above also.

Reply:
Removed frame clause from here.